Libraries/test/EventsTests.Shared/kafka-event.json

```diff
@@ -0,0 +1,33 @@
+﻿{
+  "eventSource": "aws:SelfManagedKafka",
+  "bootstrapServers": "b-2.demo-cluster-1.a1bcde.c1.kafka.us-east-1.amazonaws.com:9092,b-1.demo-cluster-1.a1bcde.c1.kafka.us-east-1.amazonaws.com:9092",
+  "records": {
+    "mytopic-0": [
+      {
+        "topic": "mytopic",
+        "partition": "0",
```

Comment:
I'm not sure whether this has to be string or int. https://docs.aws.amazon.com/lambda/latest/dg/with-kafka.html shows it as string but https://aws.amazon.com/blogs/compute/using-self-hosted-apache-kafka-as-an-event-source-for-aws-lambda/ show it as int.

I have reached out to the team to make sure we are modeling the event object right.

Reply:
Your assumption that the `Partition` should be **integer** is [true](https://github.com/confluentinc/confluent-kafka-dotnet/blob/master/src/Confluent.Kafka/Partition.cs#L54). 
I fixed JSON.